spanner/value.go

```diff
@@ -49,13 +50,55 @@ const (
 	NumericScaleDigits = 9
 )
 
+// NumericLossHandlingMode describes the mode of how to deal with loss of
+// precision on numeric values.
```

Comment:
```suggestion
// LossOfPrecisionHandling configures how to deal with loss of precision on numeric values.
// The value of this configuration is global and will be used for all Spanner clients.
```

Reply:
I changed to `LossOfPrecisionHandlingOption`, which is the type. The actual global variable is renamed to `LossOfPrecisionHandling`.